mscore/uploadscoredialog.cpp

```diff
@@ -69,6 +69,7 @@ UploadScoreDialog::UploadScoreDialog(LoginManager* loginManager)
                            .arg("</a>"));
       QFont font = licenseHelp->font();
       font.setPointSize(8);
+      font.setItalic(true);
       licenseHelp->setFont(font);
 
```

Comment:
Fine by me, in my PR I just tried to stay as close as possible with the old layout